nilearn/datasets/atlas.py

```diff
@@ -259,7 +276,12 @@ def fetch_atlas_destrieux_2009(lateralized=True, data_dir=None, url=None,
     files_ = _fetch_files(data_dir, files, resume=resume,
                           verbose=verbose)
 
-    params = dict(maps=files_[1], labels=np.recfromcsv(files_[0]))
+    params = dict(maps=files_[1],
+                  labels=pd.read_csv(files_[0], index_col=0))
+
+    if legacy_format:
+        warnings.warn(LEGACY_FORMAT_MSG)
+        params['labels'] = params['labels'].to_records()
```

Comment:
this doesn't seem to return the same result as what it was before:
- I think we need to add `to_records(index=False)` or `index_col=0` otherwise the index generated by pandas gets added to the recarray:
```
[(0, 1, 'Superior frontal sulcus', 'DefaultB', 'DefaultA', 0.68999562, 0.18570919, 0.11688445)
 (1, 2, 'Fusiform gyrus', 'No network found', 'No network found', 0.84473252, 0.02102643, 0.13378859)
 (2, 3, 'Calcarine cortex posterior', 'VisCent', 'VisPeri', 0.60176782, 0.30206712, 0.09129732)
...
```
instead of 
```
[(1, 'Superior frontal sulcus', 'DefaultB', 'DefaultA', 0.68999562, 0.18570919, 0.11688445)
 (2, 'Fusiform gyrus', 'No network found', 'No network found', 0.84473252, 0.02102643, 0.13378859)
 (3, 'Calcarine cortex posterior', 'VisCent', 'VisPeri', 0.60176782, 0.30206712, 0.09129732)
```
- the column names are different (np.recfromcsv seems to convert to lower-case)
- the dtypes of columns that contain strings are different (object for numpy and "<U<some int>" for pandas), although that may not be a big issue

I think getting an identical result with `to_records` is harder than I anticipated, (sorry about that! :/ ), so it might actually be easier to have something like 
```
if legacy:
    labels = np.recfromcsv(labels.csv)
else:
    labels = pd.read_csv()
```

I believe this might apply to some other fetchers as well.
WDYT?

Reply:
another option is to lower-case column names and use `to_records(index=False)`